app/services/converters/product_to_journal_rows.rb

```diff
@@ -0,0 +1,22 @@
+module Converters
+  class ProductToJournalRows
+
+    attr_accessor :journal, :product, :total
+
+    def initialize(journal, product, total)
```

Comment:
Would it make sense to make total an `option` in order to put the interfaces of the two converters more in sync?